service/src/main/java/com/epam/esm/dateprovider/DateProvider.java

```diff
@@ -0,0 +1,18 @@
+package com.epam.esm.dateprovider;
+
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * This class provides a current time as String in ISO 8601 format.
+ *
+ * @since 1.0
+ */
+public class DateProvider {
+
+    public static String getNowAsString() {
+        return ZonedDateTime.now(ZoneOffset.UTC).format(DateTimeFormatter.ISO_LOCAL_DATE_TIME);
```

Comment:
More looks like as util or helper, and package 'dateProvider' is not a good decision here

Reply:
commit f7c9a59